test/e2e/internal/utils/matcher.go

```diff
@@ -48,6 +54,15 @@ func (m *Matcher) MatchErrKeyWords(keywords ...string) *Matcher {
 	return m
 }
 
+// NotMatchKeyWords guarantees that the given keywords do not match with
+// the stderr.
+func (m *Matcher) NotMatchErrKeyWords(keywords ...string) *Matcher {
```

Comment:
nit:
suggesting a new name of the func.
```suggestion
func (m *Matcher) NoMatchErrKeyWords(keywords ...string) *Matcher {
```

Reply:
Updated.